packages/controller-utils/src/siwe.ts

```diff
@@ -0,0 +1,53 @@
+import { ParsedMessage } from '@spruceid/siwe-parser';
+import { bytesToString, hexToBytes } from '@metamask/utils';
+
+const msgHexToText = (hex: string): string => {
+  try {
+    const bytes = hexToBytes(hex);
+    return bytes.length === 32 ? hex : bytesToString(bytes);
+  } catch (e) {
+    return hex;
```

Comment:
Question: When do we expect to hit this block? Maybe it's because I don't understand the context, but I'm having trouble understanding why swallowing the error is the right thing to do here.

Reply:
Sorry @Gudahtt, I think my implementation was misleading after testing locally and changing back to the original implementation which has been done in the extension here ()

Could you give me an example how to use that logger in this case?

Reply:
Great, thanks!